financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentrysuccess/ManualEntrySuccessScreen.kt

```diff
@@ -0,0 +1,263 @@
+                    style = FinancialConnectionsTheme.typography.body.copy(color = titleColor)
+                )
+            }
+            Row {
+                TableCell(text = "Transaction", titleColor)
+                TableCell(text = "Amount", titleColor)
+                TableCell(text = "Type", titleColor)
+            }
+            Divider(
+                modifier = Modifier.padding(top = 4.dp, bottom = 8.dp),
+                color = FinancialConnectionsTheme.colors.borderDefault
+            )
+            for (item in tableData) {
+                val (transaction, amount, type) = item
+                Row(Modifier.fillMaxWidth()) {
+                    TableCell(text = transaction.first, transaction.second)
+                    TableCell(text = amount.first, amount.second)
+                    TableCell(text = type.first, type.second)
+                }
+            }
+        }
+        // Bottom fade.
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(36.dp)
+                .align(Alignment.BottomCenter)
+                .background(
+                    brush = Brush.verticalGradient(
+                        colors = listOf(
+                            FinancialConnectionsTheme.colors.textWhite.copy(alpha = 0f),
+                            FinancialConnectionsTheme.colors.textWhite.copy(alpha = 1f),
+                        )
+                    )
+                )
+        )
+    }
+}
+
+@Composable
+private fun buildTableRows(
+    microdepositVerificationMethod: MicrodepositVerificationMethod,
+): List<Triple<Pair<String, Color>, Pair<String, Color>, Pair<String, Color>>> {
+    val rowColor = FinancialConnectionsTheme.colors.textPrimary
+    val highlightColor = FinancialConnectionsTheme.colors.textBrand
+    return when (microdepositVerificationMethod) {
+        MicrodepositVerificationMethod.DESCRIPTOR_CODE -> listOf(
+            Triple("SMXXXX" to highlightColor, "$0.01" to rowColor, "ACH CREDIT" to rowColor),
+            Triple("GROCERIES" to rowColor, "$56.12" to rowColor, "VISA" to rowColor)
+        )
+        MicrodepositVerificationMethod.AMOUNTS -> listOf(
+            Triple("AMTS" to rowColor, "$0.XX" to highlightColor, "ACH CREDIT" to rowColor),
+            Triple("AMTS" to rowColor, "$0.XX" to highlightColor, "ACH CREDIT" to rowColor),
+            Triple("GROCERIES" to rowColor, "$56.12" to rowColor, "VISA" to rowColor)
+        )
+        MicrodepositVerificationMethod.UNKNOWN -> error("Unknown microdeposits type")
+    }
+}
+
+@Composable
+private fun RowScope.TableCell(
+    text: String,
+    color: Color
+) {
+    Text(
+        text = text,
+        style = FinancialConnectionsTheme.typography.detailEmphasized.copy(
+            color = color
+        ),
+        modifier = Modifier
+            .padding(4.dp)
+            .weight(1f)
+    )
+}
+
+@Preview
+@Composable
+internal fun ManualEntrySuccessScreenPreviewAmount() {
```

Comment:
Are these serving the purpose for testing? Should we add some unit tests?

Reply:
These will be generating screenshot tests once CI is setup for it (waiting until the whole flow is built to configure it)